modules/lvms-about-lvmcluster-cr.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="about-lvmcluster_{context}"]
+= About the LVMCluster custom resource
+
+After you have installed {lvms}, you must create an `LVMCluster` custom resource (CR) on a worker node.
+
+You can configure the `LVMCluster` CR to perform the following actions:
+
+* Create LVM volume groups that you can use to provision persistent volume claims (PVCs).
+* Configure a list of devices that you want to add to the LVM volume groups. 
+* Configure the requirements to select the nodes on which you want to create an LVM volume group, and the thin pool configuration for the volume group.
```

Comment:
Will keep this as is. The readability scores for both the sentences are same.
Thank you for the info on the possibility of ending the sentences with a preposition. :)

Reply:
Hi sure thing. Although I actually learned today that the IBM Style guide says not to do this. I always though it was something they said you didn't have to change the sentence, but today when preparing the upcoming daily snippet material, I saw that it actually says not to update to avoid ending in a preposition:

> Do not change the word order in a sentence to avoid ending the sentence with a preposition.

This is on this page: https://www.ibm.com/docs/en/ibm-style?topic=medium-tone

So although it's not critical to change for today, but in the future, it would be best to avoid.

Reply:
Sure! Thanks for the info :)